index.ts

```diff
@@ -0,0 +1,81 @@
+import { NativeModules } from 'react-native'
+
+const { RNZoomUs } = NativeModules
+
+if (!RNZoomUs) console.error('RNZoomUs native module is not linked.')
+
+async function initialize(
+  params: { clientKey: string; clientSecret: string; domain?: string },
+  settings?: {
+    // ios only
+    disableShowVideoPreviewWhenJoinMeeting?: boolean
+  },
+) {
+  if (typeof params !== 'object') {
+    throw new Error(
+      'ZoomUs.initialize expect object param. Consider to check migration docs. ' +
+        'Check Link: https://github.com/mieszko4/react-native-zoom-us/blob/master/docs/UPGRADING.md',
+    )
+  }
+
+  if (!params.domain) params.domain = 'zoom.us'
+
+  return RNZoomUs.initialize(params, {
+    // more details inside: https://github.com/mieszko4/react-native-zoom-us/issues/28
+    disableShowVideoPreviewWhenJoinMeeting: true,
+    ...settings,
+  })
+}
+
+async function joinMeeting(params: {
+  userName: string
+  meetingNumber: string | number
+  password?: string
+  participantID?: string
+  noAudio?: boolean
+  noVideo?: boolean
+
+  // IOS only fields:
+  zak?: string
+  webinarToken?: string
+}) {
+  let { meetingNumber, noAudio = false, noVideo = false } = params
+  if (!meetingNumber) throw new Error('ZoomUs.joinMeeting requires meetingNumber')
+  if (typeof meetingNumber !== 'string') meetingNumber = meetingNumber.toString()
+
+  // without noAudio, noVideo fields SDK can stack on joining meeting room for release build
+  return RNZoomUs.joinMeeting({
+    ...params,
+    meetingNumber,
+    noAudio: !!noAudio, // required
+    noVideo: !!noVideo, // required
+  })
+}
+
+async function joinMeetingWithPassword(...params) {
+  console.warn("ZoomUs.joinMeetingWithPassword is deprecated. Use joinMeeting({ password: 'xxx', ... })")
+  return RNZoomUs.joinMeetingWithPassword(...params)
+}
+
+async function startMeeting(params: {
+  userName: string
+  meetingNumber: string | number
+  userId: string
+  userType?: number // looks like can be different for IOS and Android
```

Comment:
this is unused because `userType` is forced to be `2`. make it default as `2`

Reply:
There is default value. It is not forced.

Reply:
ok, I see it now,
I think it would be cleaner to have default values on the top.
So in this case something like:
```ts
const { userType = 2 } = params;
```
on line 74

and it would be even better to store this magic number `2` into some variable like `const DEFAULT_USER_TYPE = 2` so it is clear what meaning has `2` in zoom lib